### ISSUE TITLE
Colors don't reset correctly

### ISSUE THREAD

**Issue by mqudsi** [zeit/hyper#1197]
Running hyper 1.0.0 on Windows 10, colored terminal output (16-bit, not 256-bit color) does not function correctly. In particular, colored output causes corruption of the color buffer and all text thereafter comes out wrong.

Here is a screenshot of running `ls--color ` in `bash.exe` without any plugins or customizations to either hyper or bash:

![image](https://cloud.githubusercontent.com/assets/606923/21170722/af16a4be-c18c-11e6-94aa-42fe1ed46e43.png)

**Comment by rohithmohan** [comment]
I was having this issue recently but noticed that it only occurred on my laptop and not my desktop despite being on the same Windows 10 build.

After reading https://github.com/zeit/hyper/pull/946#issuecomment-262435823, I remembered that I had messed with the color scheme on command prompt a year ago and found that the registry values for those were different from default. Not sure if that will solve it for you but fixing the registry values, solved the issue for me.
